content/v2.0/reference/config-options.md

```diff
@@ -189,3 +207,124 @@ Tracing is disabled by default.
 ```sh
 influxd --tracing-type=log
 ```
+
+---
+
+## --vault-address
+Specifies the address of the Vault server expressed as a URL and port.
+For example: `https://127.0.0.1:8200/`.
+
+```sh
+influxd --vault-address=https://127.0.0.1:8200/
+```
+
+_You can also set this using the `VAULT_ADDR` environment variable, however
+`influxd` flags take precedence over environment variables._
+
+---
+
+## --vault-ca-cert
+Specifies the path to a PEM-encoded CA certificate file on the local disk.
```

Comment:
PEM-encoded CA certificate file (`.pem`)

Reply:
Same as above. File extension doesn't matter; just the encoding.